src/index.ts

```diff
@@ -108,6 +108,23 @@ export const createFakeEggs = ({
     word: createWordGenerator(chance),
     zip: createZipGenerator(chance),
   };
+
+  type FakeEggsCustomMethods = typeof customMethods;
+  interface FakeEggs
+    extends Omit<Chance.Chance, keyof FakeEggsCustomMethods>,
+      FakeEggsCustomMethods {}
+
+  const proxyHandler: ProxyHandler<FakeEggsCustomMethods> = {
+    get(target: Record<string, unknown>, property: string, receiver: Record<string, unknown>) {
+      if (property in target) {
+        return Reflect.get(target, property, receiver);
+      }
+
```

Comment:
If we wanted to specifically exclude any chance.js method we could also add more conditions here.